src/library_glemu.js

```diff
@@ -285,7 +285,7 @@ var LibraryGLEmulation = {
 
       function ensurePrecision(source) {
```

Comment:
now that this is fragment specific, rename? e.g. `ensureFragmentPrecision`

Reply:
I agree the name change might help, but don't feel strongly either way. I'll merge this as-is later today unless there are further comments.

Reply:
OK with me.